frontend/src/views/ProfileView.vue

```diff
@@ -2,7 +2,7 @@
 	<div class="page_box_wrapper">
 		<div class="page_box">
 			<VerticalAvatarAndUserName
-				profile_picture="/test-profile.png"
+				:profile_picture="this.user.avatar"
```

Comment:
inside of <template> thingies you don't need to reference "this"

Reply:
haah thingies